### DISCUSSION TITLE
How do I update my Github profile with my name and picture?

### DISCUSSION THREAD

**Discussion by jordanpadams** [pds-data-dictionaries/PDS-Data-Dictionaries.github.io#10] [Github Help] [answered]
Just curious how I would update my Github profile with my full name and some sort of image so people can find me on GitHub?

**Comment by pds-ops** [comment, marked as answer]
Great question! And thanks for kicking off our first discussion thread!

1. After logging into Github.com, click the top-right corner to open a drop-down menu.
2. Select [Settings from the dropdown](https://github.com/settings/profile).
3. Enter your profile info so people can know who you are 😄 

This is also where you can update other settings for your profile like:
* [Website appearance](https://github.com/settings/appearance) (did you know there is a Dark Mode?)
* [Notifications](https://github.com/settings/notifications)
* [Organizations you belong to](https://github.com/settings/organizations)
* And much more. Check back occasionally as GitHub pretty regularly updates what can be configured
